airflow/sensors/base.py

```diff
@@ -330,6 +330,12 @@ def reschedule(self):
     def get_serialized_fields(cls):
         return super().get_serialized_fields() | {"reschedule"}
 
+    def raise_failed_or_skiping_exception(self, failed_message: str, skipping_message: str = "") -> None:
```

Comment:
```suggestion
    def raise_failed_or_skiping_exception(self, *, failed_message: str, skipping_message: str = "") -> None:
```

Let’s make these keyword-only

Reply:
Sounds good 👍 Just updated it